manifests/v2/third-party/jobset/jobset_manager_config.yaml

```diff
@@ -1,3 +1,2 @@
 apiVersion: config.jobset.x-k8s.io/v1alpha1
 kind: Configuration
```

Comment:
I think just using the default configuration would be sufficient.

https://github.com/kubernetes-sigs/jobset/blob/main/config/components/manager/controller_manager_config.yaml

I guess a nil configuration should be fine but it looks a bit weird.

Reply:
Either way this is just a nit.

everything LGTM to me now.

Reply:
Yeah, I just keep this empty config, so in the future we can configure it.
I think, we can enable leader election in the future, if we want to have it by default.